README.md

```diff
@@ -1 +1,3 @@
-# HackGT6-quick-tips
\ No newline at end of file
+# HackGT6-quick-tips
+
+Welcome to HackGT's quick tips! Here we have compiled various guides for you to peruse from Github and CLI to **add other topic here**. Each guide is contained within a markdown file and contains additional resources as well. Enjoy and happy learning!
```

Comment:
"we have compiled various guides for your perusal, to get you up to speed on how to make a successful hack, regardless of your experience! Each guide is contained with a markdown file and walks you through a useful hacking concept. The guides will provide additional resources for further reading at your leisure."

Honestly we should get marketing people on this.